### PR TITLE
ENS name lookup extension

### PR DESCRIPTION
**Pull request by Yukaii**
## Description

Lookup ens domain to address, or vice versa.

## Type of change

- New extension

## Screencast

 

https://user-images.githubusercontent.com/4230968/143677855-8aee33f1-1097-4959-b665-6b8709ed9d2c.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

**Review by fe9lix — CHANGES_REQUESTED** [review]
Hey @Yukaii, thanks for the contribution. 
One thing I noticed is that initially there's some loading in the list but then no results are returned. I'd suggest to:
- either skip loading when the search bar is empty and just show an empty list straight away 
- or show some default or fallback list (if there's something that makes sense...)

**Comment by Yukaii** [comment]
@fe9lix Fixed!

File: extensions/ens-name-lookup/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "name": "ens-name-lookup",
+  "title": "ENS Name Lookup",
+  "description": "Lookup ENS name or address",
+  "icon": "command-icon.png",
+  "author": "Yukai",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Lookup ENS or address",
```
Comment (fe9lix) [inline extensions/ens-name-lookup/package.json:12]:
```suggestion
      "title": "Lookup ENS or Address",
```

**Review by fe9lix — CHANGES_REQUESTED** [review]
Thx, left another small change request (title case for the command name)

**Review by fe9lix — APPROVED** [review]

**Pull request merged**